app/start.sh

```diff
@@ -11,8 +24,6 @@ fi
 # start the application using UWSGI
 $APP $DEBUG
 
-nginx -c "/etc/nginx/nginx.conf" -g "pid $NGINX_PID;"
-
 sleep 30
 APP_PID=`ps aux | grep $APP | grep -v grep`
```

Comment:
When using the fabric model, we need to check that the NGINX process is running.